src/FSharpLint.Core/Rules/Conventions/RaiseWithTooManyArguments/FailwithBadUsage.fs

```diff
@@ -60,6 +60,9 @@ let private runner (args: AstNodeRuleParams) =
 
         error
 
+    /// Error message generated by F# compiler in place of extern declaration
```

Comment:
@webwarrior-ws nit, triple slash comments are for API documentation, so let's change this to double-slash please

Reply:
They are not exclusively for API documentation.
I use `///` for all comments directly related to bindings, functions, and types, because they show up in tooltip in Visual Studio when you hover over a value and that's convenient.
I see no harm in doing so.

Reply:
alright